packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -418,10 +390,12 @@ contract L2Genesis is Script, Artifacts {
     }
 
     function _checkDevAccountsFunded() internal view {
-        for (uint256 i; i < devAccounts.length; i++) {
-            if (devAccounts[i].balance != DEV_ACCOUNT_FUND_AMT) {
+        for (uint256 i; i < cfg.getDevAccounts().length; i++) {
```

Comment:
This is doing the same thing where it calls twice per loop iteration, its using gas that it doesn't need to. Sure its a script but also we dont want to add extra compute for no reason, it will slow down tests and such

Reply:
Sorry, could've sworn I updated all these